package.json

```diff
@@ -28,50 +28,48 @@
     "commander": "^2.19.0",
     "conjure-api": "^4.14.1",
     "conjure-client": "^2.4.1",
-    "fs-extra": "^5.0.0",
+    "fs-extra": "^10.1.0",
     "lodash": "^4.17.11",
-    "mkdirp": "^0.5.1",
+    "mkdirp": "^1.0.4",
     "path": "^0.12.7",
-    "semver": "^5.6.0",
-    "sls-version": "^1.1.1",
+    "semver": "^7.3.7",
+    "sls-version": "^2.1.0",
     "ts-simple-ast": "^11.3.0",
     "tslib": "^1.9.3",
     "yargs": "^11.1.0"
   },
   "devDependencies": {
     "@blueprintjs/tslint-config": "^1.6.0",
     "@types/chai": "^4.1.6",
-    "@types/commander": "2.12.2",
```

Comment:
`commander` included its own typings at some point after this, and this `@types/commander` package had a dependency on `commander: "*"` which brought in `commander@9.something`, even though it wasn't used, but it wouldn't install on node < 12.20.